freud/plot.py

```diff
@@ -537,7 +547,9 @@ def diffraction_plot(
     )
     ax_divider = make_axes_locatable(ax)
     cax = ax_divider.append_axes("right", size="7%", pad="10%")
+    # ticks are placed at top and bottom of color bar
     cb = Colorbar(cax, im)
+    cb.ax.set_yticklabels([0, N_points])
```

Comment:
The bottom bound isn't zero because this is a log scale. Also I agreed with @gaofy95's [comment](https://github.com/glotzerlab/freud-examples/pull/38#pullrequestreview-799091565) that it would be nice to have more than two tick values on the colorbar scale. I am going to push some changes to this section and then merge this PR.

Reply:
Also the top bound is `0.7 * N_points`, not `N_points`. I am just going to revert this change to the colorbar and let it be a log scale like it was before. If we want to change plotting behavior further, let's do that in a separate PR, since everything else here is coherent/consistent for changing the normalization.